src/raster/r.survey/r.survey.py

```diff
@@ -0,0 +1,683 @@
+            for i in range(len(chunks)):
+                message = "Creating \"maps zero\" for the chunk %s  of %s"
+                gscript.message(message % (str(i), str(len(chunks))))
+                # for storing maximum view angles
+                Module("r.mapcalc", expression="xxtemp_a_{p} = 0".format(p=str(i)), quiet=True)
+                # for storing number of points a pixel is visible from
+                Module("r.mapcalc", expression="xxtemp_b_{p} = 0".format(p=str(i)), quiet=True)
+                # for storing the category of the point a pixel is visible with the maximum angle
+                Module("r.mapcalc", expression="xxtemp_c_{p} = 0".format(p=str(i)), quiet=True)
+                # for storing the minimum 3ddistance
+                Module("r.mapcalc", expression="xxtemp_e_{p} = 0".format(p=str(i)), quiet=True)
+                # for storing the maximum solid angle
+                Module("r.mapcalc", expression="xxtemp_f_{p} = 0".format(p=str(i)), quiet=True)
+                # for storing the category of the point a pixel is visible with the maximum solid angle
+                Module("r.mapcalc", expression="xxtemp_g_{p} = 0".format(p=str(i)), quiet=True)
+                # for storing the category of the point a pixel is visible with the minimum distance
+                Module("r.mapcalc", expression="xxtemp_h_{p} = 0".format(p=str(i)), quiet=True)
+                # needed for calculating the category of the point a pixel is visible with the maximum angle
+                Module("r.mapcalc", expression="xxmaxangle_{p} = 0".format(p=str(i)), quiet=True)
+                # needed for calculating the category of the point a pixel is visible with the minimum distance
+                Module("r.mapcalc", expression="xxmin3ddistance_{p} = 0".format(p=str(i)), quiet=True)
+                # needed for calculating the category of the point a pixel is visible with the maximum solid angle
+                Module("r.mapcalc", expression="xxmaxsangle_{p} = 0".format(p=str(i)), quiet=True)
+            # creating the pool for the combination parallel processes
+            pool2 = multiprocessing.Pool(nprocs)
+            # running the "collectresults" function in parallel
+            pool2.starmap(collectresults, [(t) for t in tasks2])
+            # terminating the parallel computation
+            pool2.close()
+        except:
+            # cleaning termporary layers
+            cleanup()
+            message = " ******** Something went wrong during the entire process of collection of results: please try to reduce the number of CPU (parameter 'procs') ******* "
+            gscript.error(message)
+            sys.exit()
+        else:
+            # creating the "zeros" map to be used for the FINAL combination of the different teporary maps (THE FOLLOWING CAN BE DONE CREATING A FIRTS MAP AND THEN COPYING IT, FASTENING THE PROCESS)
+            message = "Creating Final \"zeros map \" "
```

Comment:
Run Black on your code, it will flip quoting style in these strings to avoid backslashes.

Reply:
Done: We run black according to your suggestion but we also removed the backslashes manually.